src/SceneKit/SCNEnums.cs

```diff
@@ -14,8 +14,13 @@
 
 namespace SceneKit {
 
+#if NET
```

Comment:
This enums file does not appear inside SCENEKIT_API_SOURCES in src/frameworks.sources but this file also does not have the versioning using statement. Why does it compile? What's going on here?

Reply:
This file was deleted by Rolf in main since the code was dead!